app/views/govuk_publishing_components/components/docs/button.yml

```diff
@@ -147,3 +147,8 @@ examples:
     data:
       text: Button
       aria_describedby: with_aria_describedby
+  with_form_id:
+    description: Buttons will usually be contained within their containing form, but sometimes it is desirable to have the button outside the form with the `form` attribute pointing to the ID of the form
```

Comment:
Suggestion: I'd not used this attribute before and had to dig a bit to find information, worth adding a link maybe? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#form